next-netflix/app/components/Navigation/NavItem.tsx

```diff
@@ -0,0 +1,44 @@
+import styled from 'styled-components';
+import Image from 'next/image';
+import {useRecoilState} from 'recoil';
+import { activeIcon } from '../../recoil';
+
+const NavItem = ({icon}: {icon: string}) =>{
+    const [active, setActivePage] = useRecoilState(activeIcon);
+    const activeIcons = (e: React.MouseEvent<HTMLButtonElement>) =>{
+        setActivePage(icon);
+    }
+    return(
+        <NavIconBox onClick = {activeIcons}>
+            <IconImg src={`/icons/${icon}.png`} activeIcon={active} nowIcon = {icon}/>
```

Comment:
`nowIcon`을 `activeIcon`과 비교하기 위해서 넘겨주는 것이라면, 
```suggestion
            <IconImg src={`/icons/${icon}.png`} activeIcon={active === icon ? 1 : 0}/>
```
과 같이 넣어주는것이 직관적이고 props를 줄이는 방법일 것 같아요!

Reply:
오오오오 그렇네요 여긴 넘겨서 이미지 넘겨올 것도 없으니까,, 넘길때부터 삼항연산자로 넘기기! 오오 개안한 넊임 새로운 관점 쌍따봉